idl/docs/imaging.md

```diff
@@ -1,30 +1,35 @@
 Imaging
 =======
```

Comment:
At the very start, I'd remind the user that they need to run the setup script before doing any of this, as many users will forget.